tests/run.sh

```diff
@@ -161,9 +161,9 @@ wait_until_success "setup xrdb" "printf 'Xft.dpi: 96
 # Use a separate D-Bus session; sets $DBUS_SESSION_BUS_PID.
 eval "$(DISPLAY="$D" dbus-launch --sh-syntax --exit-with-session)"
 
-RC_FILE=${source_dir}/awesomerc.lua
-export AWESOME_THEMES_PATH="$source_dir/themes"
-export AWESOME_ICON_PATH="$source_dir/icons"
+RC_FILE=${AWESOME_RC_FILE:-${source_dir}/awesomerc.lua}
+AWESOME_THEMES_PATH="${AWESOME_THEMES_PATH:-$source_dir/themes}"
+AWESOME_ICON_PATH="${AWESOME_ICON_PATH:-$source_dir/icons}"
```

Comment:
Why?
I think it is better to not export if it is not necessary.
Makes it clearer that it is also used here.